docker-compose.yml

```diff
@@ -48,16 +48,30 @@ services:
       - /etc/localtime:/etc/localtime
       - /var/log/multi-cloud:/var/log/multi-cloud
 
+  redis:
+    image: redis
+    container_name: redis
+    command: redis-server
+    ports:
+      - "6379:6379"
+
   s3:
     image: opensdsio/multi-cloud-s3
     environment:
       - MICRO_REGISTRY=mdns
-      - DB_HOST=datastore:27017
+      #- DB_HOST=datastore:27017
+      - DB_HOST=tidb:4000

```

Comment:
does this need to be an install time choice ? If I am NOT using CEPH or YIG, why do i need TIDB ?

Reply:
thanks @rhsakarpos for your review, according to the former discussion, multi-cloud s3 will use tidb, so in docker-compose.yml we use tidb for testing.